analyzers/.sonarlint/sonaranalyzer-dotnetCSharp.ruleset

```diff
@@ -298,7 +298,7 @@
     <Rule Id="S4015" Action="Warning" />
     <Rule Id="S4016" Action="None" />
     <Rule Id="S4017" Action="Warning" />
-    <Rule Id="S4018" Action="Info" />
+    <Rule Id="S4018" Action="None" />
```

Comment:
S4018 - All type parameters should be used in the parameter list to enable type inference